tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -58,7 +59,8 @@ def run(self):
             )
 
         all_ssh_jobs_done = False
-        while not all_ssh_jobs_done:
+        count = 0
```

Comment:
I agree with you, but whats your take on  

https://github.com/Tendrl/commons/blob/develop/tendrl/commons/flows/authorize_ssh_key/__init__.py

"In this flow check if the ssh key is actually present in the authorized_keys file and also try to ssh on the current node using the newly added ssh key"